### PR TITLE
refact: add 3 SEC issues & enhance the intro/perf doc

### PR DESCRIPTION
**Pull request by imbajin**
## Purpose of the PR

- add 3 SEC/CVE related issues
- add the .editorconfig for devs
- enhance some high frequency reading of documents

### CODE REVIEW

File: content/cn/docs/performance/api-preformance/hugegraph-api-0.5.6-Cassandra.md [hunk 1]
```diff
@@ -64,7 +69,7 @@ weight: 2
 
 ####### 结论：
```
Comment (VGalaxies) [inline content/cn/docs/performance/api-preformance/hugegraph-api-0.5.6-Cassandra.md:70]: It seems that 7 consecutive '#' characters cannot be rendered properly:

image

File: content/cn/docs/performance/hugegraph-loader-performance.md [hunk 2]
```diff
@@ -4,19 +4,25 @@ linkTitle: "HugeGraph-Loader 性能"
...
+> **Note:** 
+> 
+> 当前的性能指标测试基于很早期的版本。**最新版本**在性能和功能上都有显著的改进。我们鼓励您参考最新的发布版本，
+> 该版本具有**自主分布式存储**和**增强的计算推下能力**。或者，您可以等待社区更新相关测试数据 (也欢迎反馈共建)。
+
 ## 使用场景
 
-当要批量插入的图数据（包括顶点和边）条数为billion级别及以下，或者总数据量小于TB时，可以采用[HugeGraph-Loader](/docs/quickstart/hugegraph-loader)工具持续、高速导入图数据
+当要批量插入的图数据（包括顶点和边）条数为 billion 级别及以下，或者总数据量小于 TB 时，
+可以采用[HugeGraph-Loader](/docs/quickstart/hugegraph-loader)工具持续、高速导入图数据
```
Comment (VGalaxies) [inline content/cn/docs/performance/hugegraph-loader-performance.md:15]: Add a space between Chinese and English.

image

File: content/cn/docs/performance/hugegraph-benchmark-0.5.6.md [hunk 3]
```diff
@@ -42,11 +47,11 @@ weight: 1
 
   - [LiveJournal dataset](http://snap.stanford.edu/data/com-LiveJournal.html)
 
-- CW使用[LFR-Benchmark generator](https://sites.google.com/site/andrealancichinetti/files)生成的人造数据
+- CW 使用[LFR-Benchmark generator](https://sites.google.com/site/andrealancichinetti/files)生成的人造数据
```
Comment (VGalaxies) [inline content/cn/docs/performance/hugegraph-benchmark-0.5.6.md:50]: ditto

**Review by VGalaxies — APPROVED** [review]

File: content/cn/docs/performance/hugegraph-benchmark-0.5.6.md [hunk 4]
```diff
@@ -16,23 +21,23 @@ weight: 1
 
 ##### 1.2.1 测试用例
 
-测试使用[graphdb-benchmark](https://github.com/socialsensor/graphdb-benchmarks)，一个图数据库测试集。该测试集主要包含4类测试：
+测试使用[graphdb-benchmark](https://github.com/socialsensor/graphdb-benchmarks)，一个图数据库测试集。该测试集主要包含 4 类测试：
```
[the remark this answers is not in the document]
Comment (imbajin) [reply content/cn/docs/performance/hugegraph-benchmark-0.5.6.md:24]: ignore legacy doc format now :)

**Pull request merged**